src/modules/site-v2/templates/tools/pairwise_indel_finder/submit.html

```diff
@@ -135,7 +135,7 @@ <h1 class="modal-title fs-5" id="generate-primers-modal-label">Thank You</h1>
 let strains;
 
 /* IGV Browser */
-const browser_div = $("#browser");
+const browser_div = $("#browser")[0];
```

Comment:
semantics -- consider calling this `browser_element` (or `browser_domElement`) instead of implementation specific detail `_div`. 
Also, whenever possible, we should minimize dependencies on jQuery (`$`). In this case, (as long as js is running after the DOM is loaded), we can use the HTML5 equivalent, `document.body.querySelector(...)` (preferred).